allennlp/tests/training/learning_rate_schedulers/learning_rate_scheduler_test.py

```diff
@@ -8,19 +9,19 @@
 
 
 class LearningRateSchedulersTest(AllenNlpTestCase):
-    def setUp(self):
-        super().setUp()
+    def setup_method(self):
+        super().setup_method()
         self.model = torch.nn.Sequential(torch.nn.Linear(10, 10))
 
     def test_reduce_on_plateau_error_throw_when_no_metrics_exist(self):
-        with self.assertRaises(ConfigurationError) as context:
+        with pytest.raises(ConfigurationError) as context:
```

Comment:
I think this should just be something like `match="learning rate ..."`, instead of `as context`, right?

Reply:
Good point. Just updated